.github/workflows/man-page-build-install.yml

```diff
@@ -0,0 +1,72 @@
+name: Man Page Build and Install
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  man-pages-build-install:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout valkey-doc
+      uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      with:
+        path: valkey-doc
+
+    - name: Checkout valkey
+      uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      with:
+        repository: valkey-io/valkey
+        path: valkey
+
+    - name: Checkout valkey-bloom
+      uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      with:
+        repository: valkey-io/valkey-bloom
+        path: valkey-bloom
+
+    - name: Checkout valkey-json
+      uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      with:
+        repository: valkey-io/valkey-json
+        path: valkey-json
+
+    - name: Set up Python
+      uses: actions/setup-python@7f4fc3e22c37d6ff65e88745f38bd3157c663f7c # v4.9.1
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y make pandoc python3-pip
+        pip3 install pyyaml
+
+    - name: Build and install man pages
+      working-directory: valkey-doc
+      run: |
+        make VALKEY_ROOT=$GITHUB_WORKSPACE/valkey \
+             VALKEY_BLOOM_ROOT=$GITHUB_WORKSPACE/valkey-bloom \
+             VALKEY_JSON_ROOT=$GITHUB_WORKSPACE/valkey-json \
+              -j;
+
+    - name: Verify man pages build
+      working-directory: valkey-doc
+      run: |
+        if [ ! -d "_build/man" ]; then
+          echo "Error: Man pages directory '_build/man' not found!"
+          exit 1
+        fi
+        
+        # Check if the directory contains any files
+        if [ -z "$(ls -A _build/man)" ]; then
+          echo "Error: No man pages were generated in '_build/man'!"
+          exit 1
+        fi
+        echo "Man pages successfully built in _build/man/"
+
+    - name: Install man pages
+      working-directory: valkey-doc
+      run: |
+        sudo make install INSTALL_MAN_DIR=/usr/local/share/man
```

Comment:
@zuiderkwast Is there anything else you would like to validate to make sure we don't silently break the man page build?

Reply:
I don't know. Checking that it can be built and installed is a good start.

I just wish it would be used more by distros and included when you install valkey. AFAIK only Fedora packages valkey-doc and users need to install it separately.

Reply:
Adoption always takes time ¯\_(ツ)_/¯